Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -100,6 +100,14 @@ class SmartPage_Context
   private _statusMessageWindow: Nullable<HTMLElement> = null;
   private _hasUnloaded: boolean = false;
 
+  /**
+   * In Firefox, the load event will be triggered right after the postback, which is not the
+   * case in other browsers and causes logic bugs. As such, we manually disable the load event
+   * for a short amount of time right after a postback happens. The onload event should already
+   * be queued by the browser due to the post back so we can use 0ms delay for the timeout.
+   */
+  private _ignoreOnLoadEventRightAfterPostBack: boolean = false;
```

Comment:
```suggestion
  private _ignoreOnLoadEventTriggeredWileProcessingThePostBack: boolean = false;
```

Or "immediately" instead of "right"